packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/updateFilling.st

```diff
@@ -0,0 +1,14 @@
+geometry
+updateFilling
+	"Calculates size of the circle representing filling state by using a logistic function.
+	If the level is below 0, the circle's radius is 0.
+	The size is set and the circle is centered in the node afterwards."
+	
+	|zeroOffset steepness level diameter|	
+
+	steepness := 0.15.
+	zeroOffset := 35.
+	self level < 0 ifTrue: [level := 0] ifFalse: [level := self level].
```

Comment:
Could you move the clamping to the `level:` method to prevent the direct usage of instance variables outside the setter?